db.go

```diff
@@ -207,11 +205,9 @@ func (db *db) upgrade(version *version) {
 	version.setState(versionActive)
 
 	if version.stats != nil {
-		title := fmt.Sprintf("A new version is active in %s.", db.name)
-		text := fmt.Sprintf("Version %s has been set to active in db %s.", version.name, db.name)
-		event := statsd.NewEvent(title, text)
-		event.Tags = []string{fmt.Sprintf("sequins_db:%s", db.name)}
-		version.stats.Event(event)
+		duration := version.active.Sub(version.created)
+		tags := []string{fmt.Sprintf("sequins_db:%s", db.name), fmt.Sprintf("version:%s", version.name)}
```

Comment:
@scottjab-stripe Do you think that by including the version, the cardinality is too high for datadog?

Reply:
I do, we got turned off about a year ago when I tried this. We do emit a log line when a version becomes active, maybe we could emit more stats that are parseable when that happens.